app/src/main/java/com/sadashi/bitrise/sample/MainActivity.kt

```diff
@@ -1,12 +1,15 @@
 package com.sadashi.bitrise.sample
 
 import android.os.Bundle
+import android.util.Log
 import androidx.appcompat.app.AppCompatActivity
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        Log.d("abcdefghijklmnopqrstuvwxyz", "")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sadashi-ota/android-bitrise-sample/blob/20d5fb5cdcee43afdd2079c85f066f299bab1eec/app/src/main/java/com/sadashi/bitrise/sample/MainActivity.kt#L13"></span>The logging tag can be at most 23 characters, was 26 (abcdefghijklmnopqrstuvwxyz)</td>
    </tr>
  </tbody>
</table>